CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 
 # These owners will be the default owners for everything in the repo.
 #*       @defunkt
-*       @pjpegion @climbfuji
+*       @pjpegion
```

Comment:
This change will remove me from the list of CODEOWNERS, so I won't be added as reviewer automatically in the future.

Reply:
thanks